projects/flow/src/lib/plugins/arrangements.spec.ts

```diff
@@ -61,7 +68,7 @@ describe('Arrangements2', () => {
       '7': { x: 660, y: 600, id: '7', deps: ['5'] },
       '8': { x: 660, y: 900, id: '8', deps: ['6', '7'] },
     };
-    const actual = Object.fromEntries(arrangements.autoArrange());
+    const actual = Object.fromEntries(arrangements._autoArrange());
```

Comment:
same her lets remove _ from functions

Reply:
Just for testing purpose made it public and added _

Reply:
yes it should be testes with public methods, call the public methods and check